index.html

```diff
@@ -6,8 +6,11 @@
     <!-- <script src="https://www.w3.org/Tools/respec/respec-w3c" async class="remove"></script> -->
     <script class="remove">
       var respecConfig = {
-        specStatus: "PRE-DRAFT",
+        previousPublishDate: "2020-11-20",
+        previousMaturity: "unofficial",
+        specStatus: "unofficial",
```

Comment:
See https://github.com/w3c-ccg/community/issues/171

It would look like thiss:
<img width="628" alt="Screen Shot 2021-01-17 at 9 24 43 AM" src="https://user-images.githubusercontent.com/8295856/104847493-f3ea5180-58a5-11eb-9ce2-efdf5b091930.png">

Reply:
since CCA 4.0 != https://github.com/BlockchainCommons/did-method-onion/blob/main/LICENSE

I have disabled respec and raised: https://github.com/BlockchainCommons/did-method-onion/issues/9